### PR TITLE
Add receptorctl traceroute documentation

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

**Review by resoluteCoder — APPROVED** [review]

File: docs/source/receptorctl/receptorctl_traceroute.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+----------
+traceroute
+----------
+
+.. contents::
+   :local:
+
+``receptorctl traceroute`` print the network route packets follow to Receptor nodes.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_traceroute.rst]:
```suggestion
``receptorctl traceroute`` displays the network route that packets follow to Receptor nodes.
```

File: docs/source/receptorctl/receptorctl_traceroute.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``.
+
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <<pid>``
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_traceroute.rst]:
```suggestion
``lsof -p <pid>``
```

Comment (oraNod) [reply docs/source/receptorctl/receptorctl_traceroute.rst]: Just noticed this. Checked the other docs and can see it in `ping` and `work list`. :face_with_diagonal_mouth:

Comment (thom-at-redhat) [reply docs/source/receptorctl/receptorctl_traceroute.rst]: and status, nice catch!

**Review by oraNod — APPROVED** [review]

**Pull request merged**